src/components/UI/SearchBar/index.css

```diff
@@ -13,12 +14,32 @@
   outline: none;
   background: var(--background);
   width: 80%;
+  padding-right: 2.5em;
 }
 
-.SearchBar > label {
-  color: var(--secondary);
+.clearSearchInputWrapper {
+  position: absolute;
+  top: 22%;
+  right: 0;
+  padding-right: 12%;
+  line-height: 1em;
 }
 
-.SearchBar > .close:hover {
+.clearSearchInput {
   cursor: pointer;
+  padding: 0 5px;
+  border: none;
+  font-weight: bold;
+  color: var(--text-secondary);
+  font-size: 1.4em;
+  text-decoration: none;
+  background: none;
+}
+
+.clearSearchInput:hover {
+  color: var(--text-primary);
+}
+
+.isEmpty {
```

Comment:
maybe you can do `.clearSearchInputWrapper.isEmpty` for the selector here to prevent issues, `.isEmpty` is too generic and maybe somebody tries to use that class somewhere else to mean a different style